synapse/rest/client/v1/admin.py

```diff
@@ -730,6 +730,69 @@ def on_GET(self, request, target_user_id):
         defer.returnValue((200, ret))
 
 
+class ServerNoticeRestServlet(ClientV1RestServlet):
+    PATTERNS = client_path_patterns("/admin/send_server_notice(/(?P<user_id>[^/]+))?")
+
+    def __init__(self, hs):
+        super(ServerNoticeRestServlet, self).__init__(hs)
+        self._notice_manager = hs.get_server_notices_manager()
+
+    @defer.inlineCallbacks
+    def on_POST(self, request, user_id):
+        """
+        This lets you send a server_notice to one or all users
+        Example:
+            http://localhost:8008/_matrix/client/r0/admin/send_server_notice/
+            @dest:user?access_token=admin_access_token
```

Comment:
if you drop the `access_token` (which is kinda obvious), does this fit on one line?

If not, can you indent this line more?

also, I suggest `@user:host` rather than `@dest:user`